CMakeLists.txt

```
@@ -14,6 +14,7 @@ find_package(std_msgs REQUIRED)
 rosidl_generate_interfaces(${PROJECT_NAME}
    "msg/Sentence.msg"
    "msg/Gpgga.msg"
+   "msg/Gpgst.msg"
    "msg/Gpgsa.msg"
```

To maintain alphabetical order:
```suggestion
   "msg/Gpgsa.msg"
   "msg/Gpgst.msg"
```